tests/assets/db/general/index.yaml

```diff
@@ -1,6 +1,10 @@
 trees:
-    rhel7: trees/rhel7.xml
-    rhel8: trees/rhel8.xml
+    rhel7:
+        template: trees/rhel7.xml
+        arches: .*
```

Comment:
`arches` are optional, so unless you need this to be explicit for some reason, you can omit them.